frontend/src/styles/global.css

```
@@ -20,6 +20,10 @@
 
   --font-sans:
     Pretendard, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+
+  --font-inter:
+    Inter, Pretendard, -apple-system, BlinkMacSystemFont, "Segoe UI",
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)